src/Core/Support/WordPressFileHeaders.php

```diff
@@ -21,7 +21,7 @@ public function headers(string $path, array $headers): array
             if (preg_match('/^[ \t\/*#@]*'.preg_quote($regex, '/').':(.*)$/mi', $data, $match) && $match[1]) {
                 $properties[$field] = trim(preg_replace("/\s*(?:\*\/|\?>).*/", '', $match[1]));
             } else {
-                $properties[$field] = '';
```

Comment:
@9585999 Why default to `null` instead of an empty string ? For me the `getHeader` method should return only one type of data which is a `string` if nothing is defined.

Reply:
but if either a string or an empty string is returned, then there is no point in the second expression, since the string `themosis_theme` will never return ...

https://github.com/themosis/framework/blob/806b2b7a4ff058f1a6e92c6e0f73cee42f29ba31/src/Core/ThemeManager.php#L265

Reply:
If the theme is installed by the CLI command, there is always a text domain. By default a theme text domain value is based on the theme name. Normally, the `themosis_theme` value should never be assigned. This is a fallback but perhaps the condition test is wrong.

Looking at the Null coalescing operator documentation, we should probably update the test using `empty()` instead in order to fallback to `themosis_theme`.

Reply:
@jlambe 
это как один из вариантов, я просто не думал что вы хахотите менять условие 😄

Reply:
@jlambe 

What is will be better?

```php
// 1
// -----------------------------------------------------------------------------
$textdomain = $this->parsedHeaders['text_domain'];
$textdomain = !empty($textdomain) ? $textdomain : 'themosis_theme';

// 2
// -----------------------------------------------------------------------------
$textdomain = $this->parsedHeaders['text_domain'];
$textdomain = empty($textdomain) ? 'themosis_theme' : $textdomain;

// 3
// -----------------------------------------------------------------------------
$textdomain = empty($this->parsedHeaders['text_domain']) ? 'themosis_theme' : $this->parsedHeaders['text_domain'];

// 4
// -----------------------------------------------------------------------------
$textdomain = !empty($this->parsedHeaders['text_domain']) ? $this->parsedHeaders['text_domain'] : 'themosis_theme';
```

Reply:
I'll go with the fourth one but perhaps it is still better to first check if the key `text_domain` is set before checking for the empty value:

```php
$textdomain = (isset($this->parsedHeaders['text_domain'] && ! empty($this->parsedHeaders['text_domain']))
    ? $this->parsedHeaders['text_domain']
    : 'themosis_theme';
```